### ISSUE TITLE
Include the extension policies into STR

### ISSUE THREAD

**Issue by c633** [coniks-sys/coniks-go#115]
Does the directory need to include its extension policies in the STR so the client can check that policies against its current pining policies?
E.g., the `Policies` struct could have a field `SignedPromiseType: string` to indicate what kind of promise the directory is using (e.g., `"TemporaryBinding"` or `"None"`). This check could be used to detect misconfig from the client-side (say, when the directory's `useTBs` is `true` but the client's `useTBs` is `false`).

**Comment by arlolra** [comment]
Yes

**Comment by masomel** [comment]
I agree, the STR `Policies` are the right place to indicate the server's extension settings for the client.

**Comment by c633** [comment]
> Is the idea that the client would simply change its configuration if it doesn't match the server's policy?

Maybe we should notice to users and let them open new issue instead?
